src/styles/_base.scss

```diff
@@ -4,6 +4,7 @@
 	height: 100vh;
 	z-index: 501;
 	overflow: hidden;
+	user-select: none;
```

Comment:
This does not fix the issue for me in firefox. I could not reproduce it on Chrome previously, and nor can I reproduced it now. I [checked in for the support](https://caniuse.com/#feat=user-select-none) of `user-select: none` in firefox, and shows, that it's only available as `-moz-` prefix. I added `-moz-user-select: none` manually on `.wrapper` through console and it worked. 

[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#Browser_compatibility) says that the firefox support is from `v69` onwards, which is not released yet.